jax/experimental/jax2tf/jax2tf.py

```diff
@@ -243,7 +243,10 @@ def __init__(self, trace: 'TensorFlowTrace', val: TfValOrUnit):
     else:  # Must be a numeric value
       assert core.skip_checks or _is_tfval(val), f"Non TfVal: {val}"
       aval = xla.abstractify(val)  # type: ignore
-      self.val = tf.convert_to_tensor(np.array(val, aval.dtype), dtype=aval.dtype)  # type: ignore
+      source_array_dtype = aval.dtype
```

Comment:
These can probably be written as a one-liner:
```
jnp.float32 if aval.dtype == jnp.bfloat16 else aval.dtype 
```

Reply:
Addressed in latest commit